static/elements/chromedash-callout.js

```diff
@@ -0,0 +1,136 @@
+import {LitElement, css, html} from 'lit-element';
+
+class ChromedashCallout extends LitElement {
+  static get properties() {
+    return {
+      signedin: {type: Boolean},
```

Comment:
nit: `isSignedIn`. and `targetId` the next line.

Please ignore if following Google styleguide is not necessary in this repo.

Reply:
Done.  I thought that they needed to match the attribute name.  But, they can be camel case here.